web/src/user_profile.ts

```diff
@@ -837,11 +837,17 @@ export function show_edit_bot_info_modal(user_id: number, $container: JQuery): v
     const is_active = people.is_person_active(user_id);
 
     assert(bot.is_bot);
+    // Extract short_name from email (format: {short_name}-bot@domain)
+    const email_local = bot.email.split("@")[0] ?? "";
+    const short_name = email_local.endsWith("-bot")
```

Comment:
Can this code be simplified? I think we can always except "@" and "-bot" in the email so we should write the code assuming that.